util/format/format.go

```diff
@@ -178,12 +178,10 @@ func OutputFormat(s string) string {
 	replace := map[string]string{
 		"'":  "''",
 		"\a": "\\a",
-		"\b": "\\b",
-		"\f": "\\f",
+		"\f": "f",
```

Comment:
Why do `\f` replace to `f`?

Reply:
That's what MySQL do. But I haven't check that in MySQL source code. It looks like it.
```
mysql> create table test_table (col set('a\fb'));
Query OK, 0 rows affected (0.01 sec)

mysql> show create table test_table;
+------------+----------------------------------------------------------------------------------------------------+
| Table      | Create Table                                                                                       |
+------------+----------------------------------------------------------------------------------------------------+
| test_table | CREATE TABLE `test_table` (
  `col` set('afb') DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=latin1 |
+------------+----------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)

mysql> select version();
+-----------+
| version() |
+-----------+
| 5.7.19    |
+-----------+
1 row in set (0.00 sec)

```
BTW: let `\v` replace to 'v' is the same reason.

Reply:
`\f` to `f` is not happening at the output stage, instead it is at the input stage:

```
mysql> select '\f';
Field   1:  `f`
Catalog:    `def`
Database:   ``
Table:      ``
Org_table:  ``
Type:       VAR_STRING
Collation:  utf8_general_ci (33)
Length:     3
Max_length: 1
Decimals:   31
Flags:      NOT_NULL 


+---+
| f |
+---+
| f |
+---+
1 row in set (0.00 sec)
```

as you can see, since `\f` is not a valid escaping sequence, `\` is just ignored.

Reply:
You may refer to this escaping list: https://github.com/mysql/mysql-server/blob/a533e2c786164af9bd276660b972d93649434297/sql/table.cc#L3910

Reply:
That's right, I found that too. But just one question, should I consider fix #4424 in `OutputFormat` function.

Reply:
https://dev.mysql.com/doc/refman/5.7/en/string-literals.html
\f doesn't need special escape, \f and f is the same.

Reply:
Yes you could fix it as well, just a few LOC.